src/controllers/connection.controller/rejectConnection/queries/queries.js

```diff
@@ -6,16 +6,14 @@ const getUserId = `query getUser($cognito_sub: String_comparison_exp) {
   `;
 
 const getPendingConnection = `query getConnection($user1: Int, $user2: Int) {
-    connections(where: { _or: [
-      { _and: [{user1: { _eq: $user1 }}, {user2: { _eq: $user2 }}] },
-      { _and: [{user1: { _eq: $user2 }}, {user2: { _eq: $user1 }}] }
-    ], status: { _eq: "pending"}}) {
+    connections(where: { 
+       _and: [{user1: { _eq: $user1 }}, {user2: { _eq: $user2 }}] ,
+      status: { _eq: "pending"}}) {
       user1
       user2
       status
     }
-  }
-  `;
+  }`;
 
 const deleteConnection = `mutation rejectConnection($user1: Int, $user2: Int) {
     delete_connections(where: {_or: [{_and: [{user1: {_eq: $user1}}, {user2: {_eq: $user2}}]}, {_and: [{user1: {_eq: $user2}}, {user2: {_eq: $user1}}]}], status: {_eq: "pending"}}) {
```

Comment:
Shouldn't we also remove the or condition from the delete connection query for the same reason?